metamodel.zenta

```diff
@@ -2044,160 +2328,659 @@ A role is assigned to human(s).
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="5dd27c67-067d-4e7d-aef1-7d1c14b37e57" targetConnections="43e3a119-3957-431b-8cc9-562cec64cb4d" zentaElement="e9e727a4-a121-4c8e-8c5f-bf48d5688035">
+            <bounds x="852" y="577" width="-1" height="-1"/>
+            <sourceConnection xsi:type="zenta:Connection" id="01fcefb4-1176-42c9-ae1a-220f6d572641" source="5dd27c67-067d-4e7d-aef1-7d1c14b37e57" target="9c707502-f7d0-498a-8c3c-069e3064342c" relationship="71c7c7b6-e5c9-4e45-b553-310216bb4785"/>
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="9c707502-f7d0-498a-8c3c-069e3064342c" targetConnections="01fcefb4-1176-42c9-ae1a-220f6d572641 97b2ef27-7968-48bc-968a-d00c6dfe5a29 5df84d74-c9cd-434e-a478-763bbed1ab01 7191f841-c5e7-423b-b155-7480f59129fa 6011cefb-ffbc-44f1-aed1-d60cd2f1ffcc b8f69467-b781-4e0d-b1e2-7c77c0eb7584" zentaElement="bb3fec3f-8700-4fb0-8715-40ce02aced70">
+            <bounds x="945" y="685" width="-1" height="-1"/>
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="76e31da6-e3d0-44b8-8e19-5b67d36130ce" targetConnections="647e979e-b436-4216-b484-c07cd053cd1e" zentaElement="0cd893e8-ee68-47e9-916f-a0bec9aa0fbd">
+            <bounds x="1029" y="444" width="-1" height="-1"/>
+            <sourceConnection xsi:type="zenta:Connection" id="0b44d600-d7a6-489b-8daa-2ccef88ccd49" source="76e31da6-e3d0-44b8-8e19-5b67d36130ce" target="12eeb6ff-7e88-4276-9bc3-7cdbc39a2ebf" relationship="2aedc6c3-2c18-48da-82f5-0aa2b6885498"/>
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="12eeb6ff-7e88-4276-9bc3-7cdbc39a2ebf" targetConnections="0b44d600-d7a6-489b-8daa-2ccef88ccd49" zentaElement="41d8aa5d-1afd-40f4-ac6d-2ef378bb08f6">
+            <bounds x="1064" y="584" width="-1" height="-1"/>
+            <sourceConnection xsi:type="zenta:Connection" id="97b2ef27-7968-48bc-968a-d00c6dfe5a29" source="12eeb6ff-7e88-4276-9bc3-7cdbc39a2ebf" target="9c707502-f7d0-498a-8c3c-069e3064342c" relationship="ecfdff07-9fe4-47b9-96b0-fd33d84f1fef"/>
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="282f542e-4dbc-41d0-8306-c99664eabab6" targetConnections="f9f449eb-b796-4c21-95c1-3417ead121ec" zentaElement="2220484d-4957-4c8d-8f7c-041adbfb17ba">
+            <bounds x="1272" y="457" width="-1" height="-1"/>
+            <sourceConnection xsi:type="zenta:Connection" id="c0f87a00-7bcc-4f21-bf64-9c8e562debe0" source="282f542e-4dbc-41d0-8306-c99664eabab6" target="fb1f833d-e1b4-4fc5-b54b-ae378749b0c5" relationship="f177d40b-0bc4-4224-957c-0d1f9f9f66ed"/>
+          </child>
+          <child xsi:type="zenta:DiagramObject" id="fb1f833d-e1b4-4fc5-b54b-ae378749b0c5" targetConnections="c0f87a00-7bcc-4f21-bf64-9c8e562debe0" zentaElement="904ce177-b87f-42b3-90ae-53e9791e4bdb">
+            <bounds x="1293" y="589" width="-1" height="-1"/>
+    
```

Comment:
I think, review should be handled as part of the CI. It's a quality gate of the CI pipeline, even if it's done manually.
I miss the product or product version here as outcome.

Reply:
For that we should model the management workflow which we are not sure enough of.